pandas/tests/io/formats/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+@pytest.fixture
+def read_file(datapath):
```

Comment:
Are you re-using this in many places? I'd prefer to just inline it in `expected_html`...

Reply:
Tom, I'm only using `read_file` from `expected_html` in the `to_html` tests. But the `data` directory that `read_file` uses is shared with the other format tests. Although at this time, no other format tests are using the `data` directory.

It could stay as it is and possibly limit code duplication in the future or could be inlined in to `expected_html` for this PR without an issue.

Reply:
I think it can just be part of `expected_html`